resources/public/api/conf/2.0/examples/historic_uk_fhl_period_summary_amend/consolidated_request.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Can these all be auto-formatted in IntelliJ, so the spacing etc is correct? Currently it shows up like this in the generated docs.  Could also do with a newline at the end of the last line.

Reply:
You may want to set your IntelliJ preferences to "format on save" so it happens automatically